gr-iio/CMakeLists.txt

```diff
@@ -0,0 +1,80 @@
+# Copyright 2011,2012 Free Software Foundation, Inc.
+#
+# This file is part of GNU Radio
+#
+# GNU Radio is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3, or (at your option)
+# any later version.
+#
+# GNU Radio is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with GNU Radio; see the file COPYING.  If not, write to
+# the Free Software Foundation, Inc., 51 Franklin Street,
+# Boston, MA 02110-1301, USA.
+
+########################################################################
+# Setup dependencies
+########################################################################
+include(GrBoost)
+
+find_package(IIO)
+
+set(GR_IIO_DEPENDENCIES
+  Boost_FOUND
+  ENABLE_GNURADIO_RUNTIME
+  ENABLE_GR_BLOCKS
+  ENABLE_GR_ANALOG
+  ENABLE_VOLK
+  IIO_FOUND)
+
+if (NOT LIBAD9361_FOUND STREQUAL "OFF")
```

Comment:
AD9361 should be an `option`, then do the `if` on the option result. The end result is roughly the same as what this code does, but the `option` code is more robust.

Reply:
Fixed

Reply:
thx!